firefoxThemes/options/options.js

```diff
@@ -30,6 +30,7 @@ function setCss(chosenTheme) {
   if (!chosenTheme) return;
   const {colors} = chosenTheme.definition;
   root.style.setProperty('--checkbox-color', colors.accentColor);
+  root.style.setProperty('--link-color', colors.editorAccentColor || colors.accentColor);
```

Comment:
I wasn't sure if the themes would re-hydrate from local storage, such that they would have the new key `editorAccentColor`, if it is true that when the new plugin loads, the new definitions get picked up I can remove the fall back.

Reply:
This comment https://github.com/doki-theme/doki-theme-web/pull/83#discussion_r772809939

Gave me the impression that the `waifuThemes` are persisted in local storage, If I make modifications to existing theme definitions (adding a color), will the existing stored definitions pick up the new changes? Are theme definitions being cached?

Reply:
> If I make modifications to existing theme definitions (adding a color), will the existing stored definitions pick up the new changes?

Yes
> Gave me the impression that the waifuThemes are persisted in local storage

It is, but every time the browser closes and reopens, dokithemedefinitions gets stored in the local storage again replacing the old one. See ‘startStorage()‘ & ‘WaifuTheme class‘ in resources.js for more info on this.

Reply:
> It is, but every time the browser closes and reopens

Okay, I am fairly sure that the browser doesn't have to restart to update a plugin, so this is only necessary until the user opens and closes the browser, right?

Reply:
> the browser doesn't have to restart to update a plugin

Did not write that.
> this is only necessary until the user opens and closes the browser, right?

This is what I wrote.

Reply:
I am confused. 

These are the facts that I think I am working with:

- dokithemedefinitions gets stored in the local storage.
-  When browser closes and reopens, dokithemedefinitions gets stored in the local storage again replacing the old one.

The point that I was trying to make is that for existing users of the plugin:

When the the plugin updates, if the browser doesn't restart, then the old dokithemedefinitions in local storage will have `editorAccentColor`.

Reply:
> When the the plugin updates, if the browser doesn't restart, then the old *dokithemedefinitions* in local storage will not have `editorAccentColor`.

Is the `editorAccentColor` property part of the new plugin update? Or is it apart of the old *dokithemedefinitions* one?

Reply:
The `editorAccentColor` property is part of the new plugin update.

Reply:
The extension will take notice of the `editorAccentColor` property so long as you update the *DokiThemeDefinitions.js* file before publishing.

Reply:
Got it, I removed the unnecessary code then. Thanks for adding clarity!